lib/if_rmap.c

```diff
@@ -134,9 +126,8 @@ static struct if_rmap *if_rmap_set(struct if_rmap_ctx *ctx,
 	return if_rmap;
 }
 
-static int if_rmap_unset(struct if_rmap_ctx *ctx,
-			 const char *ifname, enum if_rmap_type type,
-			 const char *routemap_name)
+int if_rmap_unset(struct if_rmap_ctx *ctx, const char *ifname,
+		  enum if_rmap_type type, const char *routemap_name)
```

Comment:
There's a change in behavior here. Previously, the `routemap_name` was the user input, and if the input was different from the actual configured route-map name, the `route-map doesn't exist` error was raised. Now it comes from the northbound config and is always correct, regardless of what the user passed in the command.

I don't think it's an important behavior so I think it's fine to change it. But let's remove this `routemap_name` argument then and the corresponding comparisons. They're a little bit confusing now.

Reply:
I've split the CLI command to have a no variant to make the route-map name optional in the no variant to match the newer relaxed behavior